hyperjumptech/monika
# Feat: Add --sentryDSN flag

# Monika Pull Request (PR)

## What feature/issue does this PR add

#1323 

## How did you implement / how did you fix it

1. Add Sentry dependencies
2. Implement captureException logic

## How to test

1. Edit any file, preferrably something that runs immediately like src/commands/monika.ts. Add throw new Error('Testing'
2. Run `npm run start -- --sentryDSN <your sentry dsn>
3. It should throw error and exit
4. Check your Sentry

![image](https://github.com/user-attachments/assets/adc2c7b6-1ff9-4158-a33b-8bcba2d6263e)
![image](https://github.com/user-attachments/assets/bda2e0dd-be2e-49bc-9143-033f07269813)
